srcjs/input_binding_fileinput.js

```diff
@@ -309,72 +309,67 @@ $.extend(fileInputBinding, {
     // This will be used only when restoring a file from a saved state.
     return 'shiny.file';
   },
-  _getZone: function(el) {
+  _zoneOf: function(el) {
     return $(el).closest("div.input-group");
   },
-  // This implements draghoverstart/draghoverend events that occur once per
-  // selector, instead of once for every child the way native
-  // dragenter/dragleave do. Inspired by https://gist.github.com/meleyal/3794126
-  _enableDraghover: function($el, ns = "") {
-    // Create an empty jQuery collection. This is a set-like data structure that
-    // jQuery normally uses to contain the results of a selection.
-    let collection = $();
-
-    // Attach a dragenter handler to $el and all of its children. When the first
-    // child is entered, trigger a draghoverstart event.
-    $el.on("dragenter.dragHover", e => {
-      if (collection.length === 0) {
-        $el.trigger("draghoverstart" + ns, e.originalEvent);
-      }
-      // Every child that has fired dragenter is added to the collection.
-      // Addition is idempotent, which accounts for elements producing dragenter
-      // multiple times.
-      collection = collection.add(e.originalEvent.target);
-    });
-
-    // Attach dragleave and drop handlers to $el and its children. Whenever a
-    // child fires either of these events, remove it from the collection.
-    $el.on("dragleave.dragHover drop.dragHover", e => {
-      collection = collection.not(e.originalEvent.target);
-      // When the collection has no elements, all of the children have been
-      // removed, and produce draghoverend event.
-      if (collection.length === 0) {
-        $el.trigger("draghoverend" + ns, e.originalEvent);
-      }
-    });
-  },
-  _disableDraghover: function($el) {
-    $el.off(".dragHover");
-  },
-  _enableDocumentEvents: function() {
-    let $doc = $("html");
-
-    this._enableDraghover($doc);
-    $doc.on({
-      "draghoverstart.fileDrag": e => {
-        $fileInputs.trigger("showZone.fileDrag");
+  _enableDraghover: function(el) {
```

Comment:
I think it's important to add a comment here (or at least restore the old one) indicating why `_enableDraghover` is necessary in the first place--it wasn't obvious to me at all.

Reply:
Added explanatory comment in 87f16e3c3bbb5e55edaed4881cd820665412f5b8